doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst

```diff
@@ -579,6 +579,31 @@ Bluetooth buffers configuration introduced by the :kconfig:option:`CONFIG_NCS_SA
    :start-after: fota_upgrades_zigbee_start
    :end-before: fota_upgrades_zigbee_end
 
+.. _ug_nrf5340_image_slot_numbering:
+
+Image IDs and slot numbering
+****************************
+
+Each image in the DFU (Device Firmware Update) process on the nRF5340 DK is identified by a unique ID, corresponding to a specific memory slot for each core.
+This configuration typically follows:
+
++--------+------------------------------------------------------------+-----------+-------+
+| Slot   | Description                                                | DTS label | n     |
++========+============================================================+===========+=======+
+| slot-1 | First image's primary slot (internal flash)                | "image-0" | 1     |
++--------+------------------------------------------------------------+-----------+-------+
+| slot-2 | First image's secondary slot (internal or external flash)  | "image-1" | 0, 1  |
++--------+------------------------------------------------------------+-----------+-------+
+| slot-3 | Second image's primary slot (internal RAM)                 | "image-2" | 2     |
++--------+------------------------------------------------------------+-----------+-------+
+| slot-4 | Second image's secondary slot (internal or external flash) | "image-3" | 3     |
++--------+------------------------------------------------------------+-----------+-------+
+
+.. note::
+   ``-n 0`` is the default upload target for the first image's secondary slot, if there is no other explicit selection.
+
+For the nRF5340 DK, serial recovery requires specifying ``-n 3``, indicating that updates to the network core are mediated through the application core's RAM.
```

Comment:
So, this is not a requirement of the nrf5340 but ,due to how the mcumgr-cli worked, nrf5340 in multi-image configuration with enabled `CONFIG_MCUBOOT_SERIAL_DIRECT_IMAGE_UPLOAD` will use image number as target slot number.
There is also a difference when uploading image via DFU and via recovery:
 - the DFU defaults, when 0 or no image number is given, to upload to secondary slot of first image, which is slot-1;
 -  the MCUboot default, when 0 or no image number is given, to upload to primary slot of first image.

Reply:
Thank you for clarifying - can you please check if this is now addressed correctly?